src/states.js

```diff
@@ -54,7 +54,7 @@
  **********************************************************************************************************************/
 
 var httpPort = 8080;
-var defaultHttpPort = '8080';
+var defaultHttpPort = 8080;
```

Comment:
I think this is safe to do? Seems better to keep all ports as numbers

Reply:
👍